templates/repo/projects/view.tmpl

```diff
@@ -78,7 +78,6 @@
 					{{if and $.CanWriteProjects (not $.Repository.IsArchived) $.PageIsProjects (ne .ID 0)}}
 						<div class="ui dropdown jump item poping up right" data-variation="tiny inverted">
 							<span class="ui text">
-								<img class="ui tiny avatar image" width="24" height="24">
```

Comment:
No idea what this was for but a `<img>` without a `src` can never work so I removed it.

Reply:
That's fine. Probably some dead code :)